Changelog.md

```diff
@@ -9,6 +9,7 @@ Breaking Changes:
  * ABI Encoder: Properly pad data from calldata (``msg.data`` and external function parameters). Use ``abi.encodePacked`` for unpadded encoding.
  * Code Generator: Signed right shift uses proper arithmetic shift, i.e. rounding towards negative infinity. Warning: this may silently change the semantics of existing code!
  * Code Generator: Revert at runtime if calldata is too short or points out of bounds. This is done inside the ``ABI decoder`` and therefore also applies to ``abi.decode()``.
+ * Code Generator: Use STATICCALL for pure and view functions. This was already the case in the experimental 0.5.0 mode.
```

Comment:
Should this be preformatted? "Use STATICCALL for ``pure`` and ``view`` functions."